src/Compute/Compute/Common/ComputeTokenCredential.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Azure.Core;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+
+namespace Microsoft.Azure.Commands.Compute.Common
+{
+    public class ComputeTokenCredential : TokenCredential
+    {
+        public IAccessToken accessToken { get; set; }
```

Comment:
the interface IAccessToken is from Microsoft.Azure.Commands.Common.Authentication.Abstractions
while the accesstoken required for PageBlobClient is from Azure.Core, please verify this is expected

Reply:
expected. thank you